Unity-Technologies/com.unity.multiplayer.samples.coop
# Adding utilities asmdef.

### Description
We shouldn't add any dependencies to boss room to this asmdef. asmdefs should depend on this, not the reverse.
We should move our systems here as a first step before sharing with future samples.

### Manual testing scenarios


Tested with second project, see if we could import it as package file in packman.
Tested dependencies to and from this package, works as expected.

### Contribution checklist
